docs/resources/table.md

```diff
@@ -3,8 +3,6 @@ subcategory: "Unity Catalog"
 ---
 # databricks_table (Resource)
 
--> **Public Preview** This feature is in [Public Preview](https://docs.databricks.com/release-notes/release-types.html). Contact your Databricks representative to request access. 
```

Comment:
@alexott can we keep preview just for table? we might deprecate this one...

Reply:
done